traitsui/wx/ui_panel.py

```diff
@@ -765,17 +726,18 @@ def create_fold_for_items(self, window, content):
         """ Adds a set of groups or items as vertical notebook pages to a
             vertical notebook.
         """
-        from themed_vertical_notebook import ThemedVerticalNotebook
+        raise NotImplementedError('VFold is not implemented for Wx backend')
+        #from themed_vertical_notebook import ThemedVerticalNotebook
```

Comment:
Please can we either remove this or add an explanation about why the commented-out code is still here?